src/t_list.c

```diff
@@ -1207,3 +1308,13 @@ void lmpopCommand(client *c) {
 void blmpopCommand(client *c) {
     lmpopGenericCommand(c, 2, 1);
 }
+
+/* LMMOVE destination numkeys [srckey1 ...] LEFT|RIGHT LEFT|RIGHT */
+void lmMoveCommand(client *c){
+    lmMoveGenericCommand(c, 0);
```

Comment:
Perhaps we could adapt `lmoveGenericCommand` to support multiple keys, rather than adding a method that is essentially the same.